config/application.rb

```diff
@@ -31,6 +31,7 @@ class Application < Rails::Application
     # Custom directories with classes and modules you want to be autoloadable.
     config.autoload_paths      += %W{#{config.root}/app}
     config.autoload_once_paths += %W{#{config.root}/lib}
+    config.autoload_paths += %W{#{config.root}/lib/openid}
```

Comment:
Yep, it was a really ugly way to include files at the very start. I'll revert that, thanks for pointing it.